pkg/collector/corechecks/containers/kubelet/provider/slis/provider.go

```diff
@@ -0,0 +1,80 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubelet
+
+// Package slis is responsible for emitting the Kubelet check metrics that are
+// collected from the `/metrics/slis` endpoint.
+package slis
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/containers/kubelet/common"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/containers/kubelet/provider/prometheus"
+	"github.com/DataDog/datadog-agent/pkg/util/containers"
+	prom "github.com/DataDog/datadog-agent/pkg/util/prometheus"
+	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
+	"github.com/samber/lo"
+	"strings"
+)
```

Comment:
nit: just to follow other files' import order i think `"string"` should be moved up
```suggestion
import (
	"strings"

	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/containers/kubelet/common"
	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/containers/kubelet/provider/prometheus"
	"github.com/DataDog/datadog-agent/pkg/util/containers"
	prom "github.com/DataDog/datadog-agent/pkg/util/prometheus"
	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
	"github.com/samber/lo"
)
```

Reply:
I don't think lint is going to let me do this. Whenever I run `gofmt -s -w pkg/collector/corechecks/containers/kubelet/provider/slis` it autoformats "strings" to the bottom so I think it will fail if I force it up, right?

Reply:
I you put a space between `strings` and the other imports it should work. 
the linter order only import by group (separated by a space)

Reply:
Thanks for the clarification. Made that change.